### PR TITLE
Add connectionSecret to rabbitmqsource examples

### PR DESCRIPTION
**Pull request by ChunyiLyu**
# Changes

- 📜  Add connectionSecret to rabbitmqsource examples

/kind documentation

### CODE REVIEW

File: samples/source/500-source.yaml [hunk 1]
```diff
@@ -14,6 +14,8 @@ spec:
     secretKeyRef:
       name: rabbitmq-default-user
       key: password
+  connectionSecret:
```
Comment (gab-satchi) [inline samples/source/500-source.yaml:17]: Should [this](https://github.com/knative-sandbox/eventing-rabbitmq/tree/main/samples/source#create-the-rabbitmq-source) also be updated to reflect the connectionSecret?

**Review by gab-satchi — COMMENTED** [review]
wondering if connectionSecret should be referenced elsewhere in the docs

File: samples/source/500-source.yaml (hunk 1 above)
Comment (ChunyiLyu) [reply samples/source/500-source.yaml:17]: You are totally right. I should update the step to add key `uri` to the connection secret as well.

**Review by gab-satchi — APPROVED** [review]
/lgtm

**Review by gabo1208 — APPROVED** [review]

**Pull request merged**